trivup/apps/SslApp.py

```diff
@@ -199,17 +220,76 @@ def create_cert(self, cn):
         self.exec_cmd('openssl rsa -outform der -passin "pass:%s" -in "%s" -out "%s"' %  # noqa: E501
                       (password, ret['priv']['pem'], ret['priv']['der']))
 
+        self._export_pkcs12(
+            ret, cn, through_intermediate=through_intermediate, with_ca=with_ca)
+
+        return ret
+
+    def _generate_intermediate(self, ret, cn, with_ca):
```

Comment:
I think it is better if it would return a dict instead of passing it the return dict of another function, and then the caller can do:
```python
ret.update(self._generate_intermediate(cn, with_ca))
```

Reply:
This is a good point, I've pushed an update to